### ISSUE TITLE
Multiple item Settings are not available on Android

### ISSUE THREAD

**Issue by Booklight12** [godotengine/godot#76637]
### Godot version

4.x

### System information

Android

### Issue description

The bug covers a wide range of issues, including the fact that you can't adjust debug window mode or size on Android, both of which are disabled, Forward+ rendering won't work, TAA, version control and volume fog won't work. In that case, why not just move it out like Android Export? Also, I don't know how many debugging Settings are useless on Android (and there are a lot of them) and they need a lot of testing to make sure they work on Android.

### Steps to reproduce

null

### Minimal reproduction project

null

**Comment by Calinou** [comment]
> including the fact that you can't adjust debug window mode or size on Android, 

While floating windows exist, mobile OSes do not have a concept of window mode or size **that applications can adjust themselves**. Window mode and size is entirely left to the user running the application.

> Forward+ rendering won't work, TAA, version control and volume fog won't work. 

These are expected not to work on mobile, since TAA and volumetric fog aren't implemented in the Forward Mobile rendering method (and won't be due to performance requirements).

**Comment by Zireael07** [comment]
> These settings are not hidden entirely because you may want to enable them while on Android, commit your project to version control then have TAA/volumetric fog enabled on desktop.

I think another solution is needed for this case... because having settings that are expected to NOT work is a... weird thing to say the least

**Comment by Booklight12** [comment]
That's true... The recent update of godot makes me feel a little confused. Some desired functions have been deleted and some unusable functions have been retained in the Settings list. I wonder what functions will be changed officially for the next version. (I really hope to add the function of mobile terminal export, an engine can not export, can also be called an engine? By the way... It seems that some mobile games support TAA recently, which may have something to do with their underlying optimization.)
![Screenshot_2023-05-03-01-43-54-864_com miHoYo hkrpg](https://user-images.githubusercontent.com/64678717/235745651-620e1cc6-aa7a-4516-9c11-116953b6d8f8.jpg)

**Comment by Booklight12** [comment]
That's true... The recent update of godot makes me feel a little confused. Some desired functions have been deleted and some unusable functions have been retained in the Settings list. I wonder what functions will be changed officially for the next version. (I really hope to add the function of mobile terminal export, an engine can not export, can also be called an engine? By the way... It seems that some mobile games support TAA recently, which may have something to do with their underlying optimization.)
![IMG_20230503_015410](https://user-images.githubusercontent.com/64678717/235745941-3cb7c844-8cd8-409a-b9be-d53e23153dd1.jpg)

**Comment by Booklight12** [comment]
That's true... The recent update of godot makes me feel a little confused. Some desired functions have been deleted and some unusable functions have been retained in the Settings list. I wonder what functions will be changed officially for the next version. (I really hope to add the function of mobile terminal export, an engine can not export, can also be called an engine? By the way... It seems that some mobile games support TAA recently, which may have something to do with their underlying optimization.)
![IMG_20230503_015410](https://user-images.githubusercontent.com/64678717/235746829-456db9b4-ec55-4286-91b3-75ecaa6ecaf7.jpg)

**Comment by Zireael07** [comment]
Why close? It hasn't been solved in any way yet

**Comment by Booklight12** [comment]
Because it seems like I just need to adjust the engine renderer override option on the phone like this, and I can see that the engine is not reporting any errors
![Screenshot_2023-05-06-23-08-35-227_org godotengine editor v4](https://user-images.githubusercontent.com/64678717/236632422-191d8070-1c75-4a44-b1cc-a7f659f8f538.jpg)
![Screenshot_2023-05-06-23-08-31-216_org godotengine editor v4](https://user-images.githubusercontent.com/64678717/236632425-ce559093-de70-49da-bb9a-1bd681e53fb5.jpg)
![Screenshot_2023-05-06-23-08-03-523_org godotengine editor v4](https://user-images.githubusercontent.com/64678717/236632427-7459c30c-5028-4cc6-8835-17bed851e97a.jpg)
![Screenshot_2023-05-06-23-07-19-262_org godotengine editor v4](https://user-images.githubusercontent.com/64678717/236632430-f02e74a5-0b92-41b6-aa41-40cf17e52de9.jpg)

**Comment by Booklight12** [comment]
Maybe I should open it again? What do you think?

**Comment by Zireael07** [comment]
The fact that you found a workaround for your specific problem doesn't change  the fact that having settings display but be expected not to work is downright weird.
